api/pkg/transformer/pipeline/json_output_op_test.go

```diff
@@ -702,7 +702,7 @@ func TestJsonOutputOp_Execute(t *testing.T) {
 					},
 				},
 			},
-			err: errors.New("table prediction_table is not declared"),
+			err: errors.New("invalid input: table prediction_table is not declared"),
```

Comment:
should we use a different message since the reason is different?

Reply:
ah this one should be invalid configuration and will be handled in compilation time